tests/integration/test_google_integration.py

```diff
@@ -0,0 +1,15 @@
+import os
+from load_env import load_env_file
+
+load_env_file('local-compose', file_string='config/local-compose.yaml')
+
+from processes.util.google_integration import get_drive_file
+
+def test_get_drive_file():
+    test_id = os.environ['EXAMPLE_FILE_ID']
```

Comment:
I'm using a test file ID in my local-compose. Could we upload a small file to test with, assuming the UUID of a drive file is otherwise anonymous (e.g. doesn't expose anything else about our account info or structure)?

Reply:
This is hard-coded now, I added a test folder & a small PDF.